src/test/bufferlist.cc

```diff
@@ -2369,6 +2369,21 @@ TEST(BufferList, splice) {
     EXPECT_EQ((unsigned)11, bl.length());
     EXPECT_EQ(0, ::memcmp("fghi", other.c_str(), other.length()));
   }
+
+  {
+    bufferlist bl1, bl;
+    std::vector<int> data(30);
+    // must use create_aligned
+    bufferptr ptr(buffer::create_aligned(16384, 4096));
+
+    bl.clear();
```

Comment:
Is this needed?  Shouldn't `bl` be clear at this point?

Just making sure I understand the test case.

Reply:
Looks like no-operation to me.

Reply:
> Is this needed? Shouldn't `bl` be clear at this point?
> 
> Just making sure I understand the test case.

Not needed, I will delete this line later.